js/src/Figure.ts

```diff
@@ -975,6 +979,7 @@ class Figure extends widgets.DOMWidgetView {
     y_padding_arr: any;
 
     private _update_requested: boolean;
+    private relayoutRequested: boolean;
```

Comment:
```typescript
private relayoutRequested: boolean = false;
```
Otherwise I believe it defaults to `undefined` (which does not really change the behavior here but I guess we'd better be explicit).